Mathlib/Algebra/Module/End.lean

```diff
@@ -57,9 +57,20 @@ def smulAddHom : R →+ M →+ M :=
 variable {R M}
 
 @[simp]
-theorem smulAddHom_apply (r : R) (x : M) : smulAddHom R M r x = r • x :=
+theorem smulAddHom_apply : smulAddHom R M r x = r • x :=
   rfl
 
+variable {x}
+
+lemma IsAddUnit.smul_left [Monoid S] [DistribMulAction S M] (hx : IsAddUnit x) (s : S) :
+    IsAddUnit (s • x) :=
+  hx.map (DistribMulAction.toAddMonoidHom M s)
```

Comment:
(just noting that there doesn't seem to be a better home for this lemma, since the only lower-down file would be `Algebra/Module/NatInt` which isn't a good fit).